tests/TestCase.php

```diff
@@ -93,19 +122,15 @@ protected function clearSolrIndex() {
     protected function setUp() {
         parent::setUp();
 
+        Opus_Search_Config::dropCached();
+        Opus_Search_Service::dropCached();
+
         $this->_clearTables();
         $this->clearSolrIndex();
-
-        $config = Zend_Registry::get('Zend_Config');
-        if (!is_null($config)) {
-            $this->config_backup = clone $config;
-        }
     }
 
-    protected function  tearDown() {
-        if (!is_null($this->config_backup)) {
-            Zend_Registry::set('Zend_Config', $this->config_backup);
-        }
+    protected function tearDown() {
+        $this->clearSolrIndex();
```

Comment:
There doesn't seem to be a test extending SimpleTestCase now. Anyway our CI-System is the measure of "all" things and you are right, the tests still run in less than 5 minutes, which is good enough for me. I don't like that all tests depend on Solr being present, but no need to change it now. I have pushed my local branch with your recent changes to GitHub and switched the build on the CI-System to that branch.

https://opus4ci.zib.de/jenkins/job/opus4framework-fast-git